core/lib/engine_http.js

```diff
@@ -720,7 +720,11 @@ HttpEngine.prototype._handleResponse = function (
     if (rawCookies) {
       context._enableCookieJar = true;
       rawCookies.forEach(function (cookieString) {
-        context._jar.setCookieSync(cookieString, url);
+        if (tough.Cookie.parse(cookieString) !== undefined) {
+          context._jar.setCookieSync(cookieString, url);
+        } else {
+          console.log('Could not parse cookieString from response header, skipping it ' + cookieString);
```

Comment:
TODO: find best way to do this

Reply:
Doesn't this way end up parsing every valid cookie twice? We should be able to catch the error thrown by `setCookieSync()` instead, right?

The error can be surfaced to the user by emitting an `error` event the way it's done elsewhere in the HTTP engine. Here for example: https://github.com/artilleryio/artillery/blob/74670ff2b3dcef341527615888e39348f5ac544c/core/lib/engine_http.js#L447

Reply:
Yeah, sound great, as failing is not remotely common.

Yeah saw the emit and also a log with a WARNING but wasn't sure.

I'll update both cases